software/py/vanilla_parser/stats_parser.py

```diff
@@ -437,7 +437,7 @@ def __print_manycore_tag_stats_tile_group_timing(self, stat_file, tag):
                                      ,(self.manycore_stat[tag]["global_ctr"])
                                      ,(self.manycore_stat[tag]["cycle_parallel_cnt"])
                                      ,(self.manycore_stat[tag]["instr_total"] / self.manycore_stat[tag]["global_ctr"])
-                                     ,(100 * self.manycore_stat[tag]["instr_total"] / self.manycore_stat[tag]["instr_total"])
+                                     ,(np.float64(100 * self.manycore_stat[tag]["instr_total"]) / self.manycore_stat[tag]["instr_total"])
```

Comment:
Is there ever a situation where we have zero instructions?  Why cast this to a double?

Reply:
Normally no. But I've been burned by thinking exactly like this many times, if you forget to put print_stats in your code, it's better to generate a stat file with all zeros instead of barking at you with an error, don't you think?